CHANGES.md

```diff
@@ -5,6 +5,7 @@
  - Removed extraneous files from the npm package by using the "files"
    whitelist in package.json instead of the .npmignore blacklist.
    (@anton-rudeshko)
+ - Throw if callback supplied to "finally" is invalid
```

Comment:
Most people don’t do this. Thanks for adding it. Be sure to credit yourself (`@you`) and put it above the most recent release so it gets rolled up in the next minor/patch.